qualang_tools/control_panel/video_mode/README.md

```diff
@@ -0,0 +1,183 @@
+# Video Mode
+
+This module performs a continuous rapid 2D scan of two sweep axes, and measures a signal at each point. The results are shown through an interactive web frontend with a live plot and controls for the sweep parameters.
+
+The video mode has been designed as a modular tool that is composed of four parts:
+
+1. The `OPXDataAcquirer` class, which is responsible for the data acquisition.
+2. The `ScanMode` class, which is responsible for how the 2D grid is traversed.
+3. The `InnerLoopAction` class, which is responsible for what QUA code is performed during each step of the scan.
+4. The `VideoMode` class, which handles interaction with the frontend.
+
+The `ScanMode` and `InnerLoopAction` classes are highly flexible and can be selected/modified to suit the specific needs of the user. For example, three different scan modes (`RasterScan`, `SpiralScan`, and `SwitchRasterScan`) are provided, which can be used to acquire data in different ways. Similarly, the `InnerLoopAction` class can be modified to perform additional actions, such as adding specific pulses prior to each measurement.
+
+## Installation
+
+First, it is necessary to install the `qualang_tools` package with the `videomode` extra packages:
+
+```bash
+pip install qualang-tools[videomode]
+```
+
+## Basic Usage
+To use the video mode, it is necessary to initialize the relevant classes and pass them to the `VideoMode` class. 
+We will go through a simple example to demonstrate the video mode. Most of the classes and functions described here have additional options which can be found in the docstrings and in the source code.
+
+If you don't have access to an OPX but still want to try the video mode, see the `Simulated Video Mode`section in `Advanced Usage`
+
+First, we assume that a `QuantumMachinesManager` is already connected with variable `qmm`, with a corresponding `qua_config` dictionary.
```

Comment:
You don't need a qua_config for the qmm

Reply:
wait, you do right? You need to be able to create a qm from qmm, which requires a qua_config right?

Reply:
The config is attached to the QM, but not the QMM so maybe it is just about the phrasing